clang/test/CodeGen/aarch64-sme-inline-streaming-attrs.c

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -S -target-feature +sme -verify -DTEST_NONE -x c %s
```

Comment:
Doing `-x c` on a .c file is redundant, as it's already interpreted as a C source file.